index.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
```

Comment:
This is per-tab: so each tab will do one pre-flight (if it calls any APIs) and will manage its own cache.
Browsers don't cache POSTs.